requirements.txt

```diff
@@ -62,3 +62,6 @@ marshmallow==2.15.3
 # License: Apache 2.0
 # Upstream url: https://pypi.org/project/responses/
 responses==0.9.0
+
+# SDK for integration with Jira
+jira==2.0.0
```

Comment:
Could we have it in `extra_require` in setup.py as this is optional feature?
Example: https://github.com/lyft/amundsenmetadatalibrary/blob/master/setup.py#L24

Reply:
sure - do all extra requires need a separate docker build? or is that specific to the oidc image because it's an auth library?

Reply:
Good point. I think this line can be changed to install this, but I think it's fine to leave it as is and let consumer decide.
https://github.com/lyft/amundsenfrontendlibrary/blob/master/public.Dockerfile#L20

If the questions was about us, I think we can choose it to install in amundsenfrontend-private.

Reply:
i kind of wonder if there's something weird going on because of how this is set up - regardless of what i try it, CI says 

`Processing /home/travis/build/lyft/amundsenfrontendlibrary
  amundsen-frontend 2.0.0 does not provide the extra 'jira'`

Reply:
i think i'm going to pass on this change for now since it's blocking CI